client/tailscale/localclient.go

```diff
@@ -933,7 +933,18 @@ func CertPair(ctx context.Context, domain string) (certPEM, keyPEM []byte, err e
 //
 // API maturity: this is considered a stable API.
 func (lc *LocalClient) CertPair(ctx context.Context, domain string) (certPEM, keyPEM []byte, err error) {
-	res, err := lc.send(ctx, "GET", "/localapi/v0/cert/"+domain+"?type=pair", 200, nil)
+	return lc.CertPairWithValidity(ctx, domain, 0)
+}
+
+// CertPair returns a cert and private key for the provided DNS domain.
+// When minValidity is non-zero, the returned certificate will be valid for
+// at least the given duration, if permitted by the CA.
+//
+// It returns a cached certificate from disk if it's still valid.
```

Comment:
Optional suggestion:
```go
// It returns a cached certificate from disk if it's still valid.
// If minValidity is non-zero, the returned certificate will be valid
// for at least the given duration, if permitted by the CA, and in
// that case a cached certificate is considered invalid if its validity
// period is less than minDuration.
```

just to make it clear how `minDuration` relates to the cache a bit more. Or words to that effect. YMMV.